src/components/APIStarter.jsx

```diff
@@ -35,9 +390,21 @@ const APIStarter = () => {
     //     );
     //   })}
     // </ul>
-    <>{ response.length==0 ? <div></div>: (<div>
-        <QuestionView question ={response[index]} index={index} setIndex={setIndex} limit={10}/>
-    </div>)}</>
+    <>
+      {response.length == 0 ? (
+        <div></div>
+      ) : (
+        <div>
+          <QuestionView
+            question={response[index]}
+            correctAns={allAnswers}
+            index={index}
+            setIndex={setIndex}
+            limit={2}
```

Comment:
Don't hardcode limit value. Have a separate state variable for this.

Reply:
Done